java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture6/tasks/_3/UserService.java

```
@@ -0,0 +1,11 @@
+package com.zoolatech.lecture6.tasks._3;
+
+record UserService(UserRepository userRepository) {
+
+    public String findUserEmail(String id) throws UserMissingException {
```

```
UserService needs to call the method of UserRepository and properly handle the thrown exception
```

the exception is not handled